src/YOUR_NAME_ex/pa02/chutes_simulation.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Kaspar Akilles Lilja, Kevin Martin Lankut'
+__email__ = 'kalilja@nmbu.no, kela@nombu.no'
+
+
+import random
+
+
+class Board:
+
+    def __init__(self, chutes=None, ladders=None, goal=None):
+        """
+        :param chutes: a custom list of chutes
+        :param ladders: a custom list of ladders
+        :param goal: custom goal where you can choose the value
+        All these parameters have set standards, but can be altered.
+        """
+        if chutes is None:
+            chutes = [(24, 5), (33, 3), (42, 30), (56, 37), (64, 27),
+                      (74, 12), (87, 70)]
+
+        if ladders is None:
+            ladders = [(1, 40), (8, 10), (36, 52), (43, 62), (49, 79),
+                       (65, 82), (68, 85)]
+
+        self.chutes_ladders = dict(chutes + ladders)
+
+        if goal is None:
+            self.goal = 90
+        else:
+            self.goal = goal
+
+    def goal_reached(self, position):
+        """
+        :param position: takes the position of a player
+        :return: returns True if the player have reached goal
+        """
+        return position >= self.goal
+
+    def position_adjustment(self, position):
+        """
+        :param position: takes the position of a player
+        :return: returns if the player should move up a ladder or down a chute
+        """
+        for start, end in self.chutes_ladders.items():
+            if position == start:
+                return end - start
+        return 0
```

Comment:
check out the following: `return self.chutes.get(position, position) - position`